src/cephadm/cephadm

```diff
@@ -3903,8 +3903,18 @@ def command_deploy(ctx):
 
     # Get and check ports explicitly required to be opened
     daemon_ports = [] # type: List[int]
-    if ctx.tcp_ports:
-        daemon_ports = list(map(int, ctx.tcp_ports.split()))
+
+    # on redeploy or reconfig, ports that are to be checked
+    # could fail check that they are not "in use" since they
+    # are being used by the previously deployed service.
+    # Is there a way to check that ports received from module
+    # have changed since past deployment (which mean they would
+    # in fact need a check)? Is there a way to know if "in use"
+    # ports are being used by the service we wish to assign them
+    # to or taken by something else?
```

Comment:
https://unix.stackexchange.com/a/238847/160469

But I'm :+1: with simply disabling the port-in-use check for re{deploy|config}

Reply:
Going to stick with just not rerunning this on on redeploy/reconfig for now. Maybe something to look into later.